mongo/test/org/immutables/mongo/bson4gson/JavaTimeTest.java

```diff
@@ -75,7 +76,7 @@ public void localDate() throws IOException {
 
   @Test
   public void localDateTime() throws IOException {
-    LocalDateTime now = LocalDateTime.now();
+    LocalDateTime now = LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS);
```

Comment:
What is the motivation for this change? Is it a java-level specific issue, or more of a tidying/improvement change?

Reply:
Between Java 8 and Java 9 Instant precision changed from millis to nanos. So on JDK9+ we get greater prescision. But when you serialise and deserialize Instants using Gson you still get millis precision.

In order to this test to work on all JDK versions, I eagerly reset precision to millis to unify all JDK builds.

Reply:
Today I learned! 

Thank you :)

Reply:
Interesting, I thought [Instant](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html) had always nano-second precision:

From javadoc: 
>  the class stores a long representing epoch-seconds and an int representing nanosecond-of-second, which will always be between 0 and 999,999,999.